packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -69,11 +71,11 @@ const ImageGridColumnsTopStory = css`
   grid-column: 1 / span 6;
 
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    grid-column: 1 / span 2;
+    grid-column: 1 / span 3;
   }
 
   @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
```

Comment:
Since we are using `grid-column: 1 / span 6;` as a default, shall we delete this media query and add a max to the previous one?

```suggestion
  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX} ) {
```

Reply:
Nice spot. Added `max-width`